tests/test_mapping.rs

```diff
@@ -66,61 +69,58 @@ impl PyMappingProtocol for Mapping {
     }
 }
 
+/// Return a dict with `m = Mapping(['1', '2', '3'])`.
+fn map_dict(py: Python) -> &pyo3::types::PyDict {
+    let d = [("Mapping", py.get_type::<Mapping>())].into_py_dict(py);
+    py_run!(py, *d, "m = Mapping(['1', '2', '3'])");
+    d
+}
```

Comment:
If it were possible to return a value from `py_run!`, I would suggest the following:

```suggestion
fn make_mapping(py: Python) -> &PyAny {
    let cls = py.get_type::<Mapping>();
    py_run!(py, cls, "cls(['1', '2', '3'])")
}
```

Use like

```
let mapping = make_mapping(py);
py_assert!(py, mapping, "mapping['1'] == 0");